.github/workflows/publish-helmcharts-to-artifactory.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: "Push Helm Charts to Artifactory"
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches: [master, v1.*]  # Running on v1.* to tag official release with branch name
+jobs:
+  build_publish_helm_charts:
+    env:
+      HELM_CHART_ARTIFACTORY_URL: "https://artifactory.magmacore.org:443/artifactory/"
+      HELM_CHART_MUSEUM_REPO: helm-test
+      HELM_CHART_MUSEUM_USERNAME: "${{ secrets.HELM_CHART_MUSEUM_USERNAME }}"
+      HELM_CHART_MUSEUM_TOKEN: "${{ secrets.HELM_CHART_MUSEUM_TOKEN }}"
+      MAGMA_ROOT: "${{ github.workspace }}"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      # Version is github job run number when running on master
+      # Or is branch name when on release branch
+      - name: Set Helm chart version
+        run: |
+          if [ "${GITHUB_REF##*/}" = "master" ] ;then
+            echo "VERSION=${GITHUB_RUN_NUMBER}" >> $GITHUB_ENV
```

Comment:
can we use ${GITHUB_SHA::8} instead?

Reply:
no helm chart version respect semantic versioning which makes it impossible to use here.